cylc/flow/dbstatecheck.py

```diff
@@ -108,6 +126,10 @@ def get_remote_point_format_compat(self):
         ):
             return row[0]
 
+    def get_point_format(self):
+        """Return the cycle point format of this DB."""
+        return self.point_fmt
+
```

Comment:
I struggle to get over historical arguments (pre-Python possibly) that getter methods should always be used. In this case, fair enough, I'll ditch the method. 

We should probably be using `@property` decorator to kind-of do read-only access. https://docs.python.org/3/library/functions.html?highlight=property#property

Reply:
> (pre-Python possibly) that getter methods should always be used

Java!

Reply:
> We should probably be using `@property` decorator to kind-of do read-only access. https://docs.python.org/3/library/functions.html?highlight=property#property

Only if it offers a reasonable advantage IMO, because it comes with a slight performance (and complexity) penalty